decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -628,6 +628,39 @@ def generate_localized_title(field = nil, skip_injection: false)
     organization
   end
 
+  factory :taxonomy, class: "Decidim::Taxonomy" do
+    name { Decidim::Faker::Localized.literal(generate(:name)) }
```

Comment:
We should try to use the XSS version, so that we make sure we do not get into trouble.
```suggestion
    transient do
      skip_injection { false }
    end
    
    name { generate_localized_title(:taxonomy_name) }
```